kikuchipy/signals/ebsd.py

```diff
@@ -881,6 +883,54 @@ def get_image_quality(self, normalize: bool = True) -> np.ndarray:
 
         return image_quality_map
 
+    def index(
```

Comment:
I think we should come up with a better, more specific method name, so we don't have to change it later if we add other indexing related methods (e.g. pattern matching like by Foden et al.) which might be confused with this. I suggest `dictionary_indexing()` although it would be great to have a name in a similar vein to the `remove_static_background()` and `get_image_quality()`, which are easier to understand the outcome of...

Reply:
I think `dictionary_indexing` is fairly clear, certainly preferable to `index`